src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -24,6 +24,27 @@ class Filesystem
 {
     private static $lastError;
 
+    public static function joinPaths(string ...$pathSegments): string
```

Comment:
A description including an example would be helpful

Reply:
Do the docs I've just created help? https://github.com/symfony/symfony-docs/pull/12803

Reply:
Sure thing but the method itself should have a description in the phpdoc too